vectorlink-worker/src/main.rs

```diff
@@ -29,7 +64,11 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         args.identity.unwrap_or_else(generate_identity),
     )
     .await?;
+    successful_connection_counter.inc();
+
     VectorlinkTaskHandler::process_queue(&mut queue).await?;
+    successful_task_counter.inc();
 
+    // wait(); // for testing; you can see the metrics at http://localhost:9184/metrics
     unreachable!();
 }
```

Comment:
I'm wondering if we have to wait for the counters to be scraped before shutting the machine down. I think the scraping happens at least every 15 seconds, but I'll do testing on this in the future.

Reply:
honestly probably it's not that important. we know what tasks completed because it is all in etcd. this is more about giving prometheus the opportunity to analyse time series and give us cool statistics that will mostly work on average, rather than a system we must absolutely 100% have reflect the truth.